apps/remix-ide/src/app/plugins/parser/code-parser.tsx

```diff
@@ -124,6 +124,10 @@ export class CodeParser extends Plugin {
 
         this.on('fileManager', 'currentFileChanged', async () => {
             await this.call('editor', 'discardLineTexts')
+            const completionSettings = await this.call('config', 'getAppParameter', 'auto-completion')
+            if (completionSettings) {
+                this.antlrService.getCurrentFileAST()
```

Comment:
if it's supposed to return something anymore, renaming the function would be better